### PR TITLE
Immediate versions of `load` and `activate`

### PR DESCRIPTION
**Pull request by cartermak**
Updates the schema to create versions of `load` and `activate` without `time` properties, renames `immediate_command` to `immediate_fsw_command`, and makes these types valid for `immediate_commands`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nasa-jpl/seq-json-schema",
-  "version": "1.0.21",
```
Comment (sqlAlchemyNoob2004) [inline package.json:3]: 😉

File: schema.json [hunk 2]
```diff
@@ -82,25 +144,25 @@
         "metadata": {
           "$ref": "#/$defs/metadata"
         },
+        "stem": {
```
Comment (goetzrrGit) [inline schema.json]: Can we keep the order the same for `stem` and `type` in the `command` object. It adds noise to the PR and I just want to focus on what we are actually changing.

**Review by goetzrrGit — CHANGES_REQUESTED** [review]
Looks good. Nothing major just fix the order of `Stem` and `Type`. @dandelany is out today but I want to get his eyes on this too as he wasn't part of the meeting.

File: package.json (hunk 1 above)
Comment (goetzrrGit) [reply package.json:3]: @cartermak We probably want to up the version to 1.0.22

Comment (sqlAlchemyNoob2004) [reply package.json:3]: [according to semver](https://semver.org/#summary), `PATCH version when you make backward compatible bug fixes`

this doesn't really fall into that category. i think minor version increment is correct.

File: schema.json (hunk 2 above)
Comment (cartermak) [reply schema.json]: Good catch; cleaned up the diff

File: package.json (hunk 1 above)
Comment (cartermak) [reply package.json:3]: Agree -- this is a backward-compatible added feature, so I think it merits a minor version increment. Sequences that leverage this added feature won't be backward compatible.

**Review by dandelany — APPROVED** [review]
👍 thanks @cartermak - this looks good to me & tests are passing. I'll coordinate with @goetzrrGit re: getting this released in the next Aerie version

**Review by goetzrrGit — APPROVED** [review]

**Pull request merged**